libs/langgraph/tests/test_pregel.py

```diff
@@ -6053,7 +6053,76 @@ def parent_call_multiple_subgraphs(inputs):
 
 
 @pytest.mark.parametrize("checkpointer_name", ALL_CHECKPOINTERS_SYNC)
-def test_multiple_subgraphs_mixed_checkpointer(
+def test_multiple_subgraphs_mixed_state_graph(
+    request: pytest.FixtureRequest, checkpointer_name: str
+) -> None:
+    checkpointer = request.getfixturevalue(f"checkpointer_{checkpointer_name}")
+
+    class State(TypedDict):
+        a: int
+        b: int
+
+    class Output(TypedDict):
+        result: int
+
+    # Define addition subgraph
+    @entrypoint()
+    def add(inputs):
+        a, b = inputs
```

Comment:
Perhaps use a dict here instead of a 2-tuple to make the test code easier to read?

Reply:
i'll add a type annotation but i think useful to test non-dict inputs